README.md

```diff
@@ -1,15 +1,26 @@
+
 # openshift-goimports
 Organizes Go imports according to OpenShift best practices
 
-## Summary
+* [Summary](#Summary)
+* [Example sorted import block](#Examplesortedimportblock)
+* [Installation](#Installation)
+* [Usage](#Usage)
+* [Examples](#Examples)
+	* [Example CLI usage](#ExampleCLIusage)
+	* [Example hack/tools.go file](#Examplehacktools.gofile)
+	* [Example hack/verify-imports.sh script](#Examplehackverify-imports.shscript)
+	* [Example Makefile sections](#ExampleMakefilesections)
```

Comment:
Suggestion using GitHub's default anchors. I'm not sure what GH will do with non-alphanumeric characters.

```suggestion
* [Summary](#summary)
* [Example sorted import block](#example-sorted-import-block)
* [Installation](#installation)
* [Usage](#usage)
* [Examples](#examples)
```